SPECS/nbdkit/nbdkit.spec

```diff
@@ -0,0 +1,2116 @@
+%global _hardened_build 1
+%ifarch %{kernel_arches}
+# ppc64le broken in rawhide:
+# https://bugzilla.redhat.com/show_bug.cgi?id=2006709
+# riscv64 tests fail with
+# qemu-system-riscv64: invalid accelerator kvm
+# qemu-system-riscv64: falling back to tcg
+# qemu-system-riscv64: unable to find CPU model 'host'
+# This seems to require changes in libguestfs and/or qemu to support
+# -cpu max or -cpu virt.
+# s390x builders can't run libguestfs
+%ifnarch %{power64} riscv64 s390 s390x
+%global have_libguestfs 1
+%endif
+%endif
```

Comment:
We can drop these bits, as they don't apply in Mariner:
1. Hardened builds are enabled by default.
2. We don't have the `kernel_arches` macro from what I was able to find.
```suggestion
```

Reply:
I have removed have_libguestfs flag from the SPEC file so that it'll always build the libguestfs plugin.